src/Event/EventBody.ts

```diff
@@ -20,12 +19,17 @@ export default class EventBody extends Engine.EventBody {
 
     createElement() {
         const flag = super.createElement();
-        const template = tinytime('{YYYY}.{Mo}', { padMonth: true });
+        // const template = tinytime('{YYYY}.{Mo}', { padMonth: true });
```

Comment:
Why did you make this line into a comment rather than delete it? Please make sure that you can justify every line of diff you submitted with a basis. Don't submit any unecessary lines of diff